resources/lang/en/layout.php

```diff
@@ -180,7 +180,7 @@
         ],
         'support' => [
```

Comment:
is this footer only shown when logged out? if so we can probably just remove this section.